fs/xfstests.py

```diff
@@ -73,17 +73,21 @@ def setUp(self):
         # enabler for older runners, but should be removed soon
         elif self.detected_distro.name in ['centos', 'fedora', 'rhel',
                                            'redhat', 'SuSE']:
-            packages.extend(['acl', 'bc', 'dump', 'indent', 'libtool', 'lvm2',
+            packages.extend(['acl', 'bc', 'dump',  'libtool', 'lvm2',
                              'xfsdump', 'psmisc', 'sed', 'libacl-devel',
                              'libattr-devel', 'libaio-devel', 'libuuid-devel',
                              'openssl-devel', 'xfsprogs-devel'])
+
+            if not (self.detected_distro.name == 'rhel' and '8' in self.detected_distro.version):
```

Comment:
Instead of adding the same package for each distribution, better to add it once and remove only in case of `rhel 8`.

Reply:
@harish-24 ah yes looks good idea